src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/AbstractPersistenceStrategy.java

```diff
@@ -172,6 +172,23 @@
 		return this.getLinkedElements(element);
 	}
 
+	/**
+	 * Get all links where the decision knowledge element is either the source or
+	 * the destination element.
+	 *
+	 * @see Link
+	 * @param element
+	 *            decision knowledge element with id in database.
+	 * @return list of links where the given decision knowledge element is either
+	 *         the source or the destination element.
+	 */
+	public List<Link> getLinks(DecisionKnowledgeElement element) {
+		List<Link> links = new ArrayList<Link>();
+		links.addAll(this.getInwardLinks(element));
```

Comment:
I get a link returned here that has no sourceElement. So this will later cause a nullPointerException.
I propose to change getInwardLinks(.) in IssueStrategy to:
`for (IssueLink inwardIssueLink : inwardIssueLinks) {
			Link link = new LinkImpl(inwardIssueLink);
			if(link.getSourceElement() != null && link.getDestinationElement() != null) {
				inwardLinks.add(link);
			}
		}`